docs/developer/centrifuge-api/overview/index.md

```diff
@@ -0,0 +1,203 @@
+---
+id: overview
+title: Overview
+category: subpage
+contributors: <Filippo:filippo@embriotech.ch>
+---
+
+# Overview
+
+Centrifuge API leverages [GraphQl](https://graphql.org/). GraphQL is an open-source query language standard for APIs and a runtime. It uses a strongly-typed approach to querying large and complex sets of data. It provides a more efficient, powerful, and flexible alternative to the traditional REST API approach. With GraphQL, clients can request exactly the data they need, and nothing more, which reduces the amount of data transferred over the network and improves performance. It also allows for more predictable and structured responses, making it easier to work with complex data structures.
```

Comment:
Can you rewrite this to more of an overview of what the API does, rather than a description of GraphQL?